rest/Endpoints.ts

```diff
@@ -363,6 +363,67 @@ export default class Endpoints {
     return `/guilds/${guildID}/templates/${code}`;
   }
 
+  // Threads
+  /**
+   * `/channels/:channelID/messages/:messageID/threads`
+   * - POST - Creates a new thread from an existing message
+   */
+  public static START_THREAD_WITH_MESSAGE(channelID: string, messageID: string) {
```

Comment:
idk how relevant this is but I might suggest:
- start_thread for this one
- create_thread for the no message one.
If this is discord naming (which I haven't checked, we can keep the names as they are, they have the advantages of being very clear and descriptive)

Reply:
(so probably disregard that, especially for endpoint, but might be worth thinking about for the according methods)

Reply:
Yes that's how Discord named them